### PR TITLE
feat: add environment info in the sentry report

### PR DESCRIPTION
**Pull request by douglasduteil**
---

+ Use https://browser.sentry-cdn.com CDN
+ Parse the `window.location` to guess the environment
+ Run prettier :sparkles:

### CODE REVIEW

**Review by revolunet — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -129,15 +137,59 @@
...
+      Fabrique numérique des Ministères Sociaux -
+      <a href="mailto:DSI-incubateur@sg.social.gouv.fr"
+        >DSI-incubateur@sg.social.gouv.fr</a
+      >
     </footer>
     <script>
-      Raven.config('https://7b80ba66958845f4b44790ffc7191638@sentry.fabrique.social.gouv.fr/28').install();
-      Raven.captureException(new Error());
+      (function () {
+        var url = new URL(location);
```
Comment (douglasduteil) [inline index.html]: @revolunet [IE11 does not support `URL` apparently :(](https://caniuse.com/#feat=url)

Comment (revolunet) [reply index.html]: haha best IE11 joke ever :)

good to know !

Comment (revolunet) [reply index.html]: document.location should do the trick ?

Comment (revolunet) [reply index.html]: i can confirm it does : `document.location.hostname` is available in IE11

Comment (douglasduteil) [reply index.html]: Change done (not tested)

**Review by revolunet — APPROVED** [review]

**Pull request merged**